recipes/dropstackframe/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "dropstackframe" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: dropstackframe
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dropstackframe-{{ version }}.tar.gz
```

Would include the `name` variable in the tarball filename as well

```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```